### PR TITLE
First version of welding flow documentation

### PR DESCRIPTION
**Pull request by fcabrera23**

### CODE REVIEW

**Review by dkirby-ms — APPROVED** [review]

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
```
[the remark this answers is not in the document]
Comment (fcabrera23) [reply docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md:1]: Added

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+type: docs
+weight: 9
+title: Welding Defect
+linkTitle: WeldingDfefect
```
[the remark this answers is not in the document]
Comment (fcabrera23) [reply docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md:5]: Fixed

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+# Jumpstart Agora - Contoso Motors Welding defect flow
```
[the remark this answers is not in the document]
Comment (fcabrera23) [reply docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md]: Fixed

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+1. **RTSP video simulation:** The welding defect flow requires a particular video of a welding to apply the AI inference. In this scenario, due to the lack of a real video camera, an RTSP simulated feed is used. The simulated feed is designed to closely mimic the behavior of a real welding scenario, providing a reliable video for the AI inference process.
+
+1. **Frame capturing:** This step involves using **OpenCV** to establish a connection with the RTSP video feed and retrieve the welding video frames. Each frame is then passed to the appropriate welding AI inference class, which applies the required preprocessing and post process to the frame. The welding AI inference class is implemented in [welding.py](https://github.com/microsoft/jumpstart-agora-apps/blob/manufacturing/contoso_manufacturing/developer/webapp-decode/welding.py) and is designed to handle the specific requirements of the welding defect detection process.
```
[the remark this answers is not in the document]
Comment (fcabrera23) [reply docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md]: Fixed

**Pull request merged**